packages/client/src/arpa_reporter/views/UploadsView.vue

```diff
@@ -6,43 +6,72 @@
       <div class="col">
         <DownloadTemplateBtn />
 
-        <router-link to="/new_upload" class="btn btn-primary ml-2">
+        <router-link
+          to="/new_upload"
+          class="btn btn-primary ml-2"
+        >
           Submit Workbook
         </router-link>
       </div>
     </div>
 
     <vue-good-table
+      ref="uploadsTable"
       :columns="columns"
       :rows="rows"
       :group-options="groupOptions"
       :sort-options="{
         enabled: true,
         initialSortBy: defaultSortOrder
       }"
-      styleClass="vgt-table table table-striped table-bordered"
-      ref="uploadsTable"
+      style-class="vgt-table table table-striped table-bordered"
```

Comment:
I can't find this syntax in the docs. I can only find the old syntax

https://xaksis.github.io/vue-good-table/guide/style-configuration/style-classes.html#style-classes

Reply:
Good question — Vue actually treats the two syntaxes as synonymous, and translates either to the same underlying prop name. The lint rule simply standardizes our usage within templates to kebab case, as is the community style. 

Bit more about this all documented here: https://vuejs.org/style-guide/rules-strongly-recommended.html#prop-name-casing

Just to double-check my understanding, I added a new class `new-test-class` into this prop, then confirmed the table was rendered with the new class: 

<img width="950" alt="Screenshot 2024-04-02 at 8 23 55 PM" src="https://github.com/usdigitalresponse/usdr-gost/assets/126250/610b180f-b26d-4723-a049-64de81f51799">
<img width="2496" alt="Screenshot 2024-04-02 at 8 23 20 PM" src="https://github.com/usdigitalresponse/usdr-gost/assets/126250/ef7037cd-8107-4baa-ab6b-969c416735c3">

Reply:
Perfect, thank you. That explanation is great. Thanks for checking.